packages/ui/src/theme/__tests__/overrides.test.ts

```diff
@@ -563,7 +563,7 @@ describe('@aws-amplify/ui', () => {
         --amplify-components-stepperfield-flex-direction: column;
         --amplify-components-stepperfield-input-text-align: center;
         --amplify-components-switchfield-disabled-opacity: var(--amplify-opacities-60);
-        --amplify-components-switchfield-focused-shadow: var(--amplify-shadows-small);
+        --amplify-components-switchfield-focused-shadow: 0px 0px 0px 2px var(--amplify-colors-border-focus);
```

Comment:
Do we also need to update the CSS variables here? https://ui.docs.amplify.aws/components/authenticator?platform=react#amplify-css-variables

Reply:
HMMM yes, I just updated this PR to include that change. Since it currently exists, we should keep it up to date. Outside of this PR, I'm wondering if:
- should we show just those css vars specific to the authenticator on this page? I wonder if it makes more sense to just show --amplify-components-authenticator-* variables and maybe a link to a full list elsewhere.
- we should have a less manual way of updating that page?